website/versions.json

```diff
@@ -1 +1 @@
-["1.17", "1.16", "1.15", "1.14", "1.13", "1.12"]
+["2.0", "1.17", "1.16", "1.15", "1.14", "1.13", "1.12"]
```

Comment:
I don't think we should publish docs for beta versions as they are subject to change, and at the same time this will duplicate work to add/update docs for 2.0

Reply:
Ok I removed the doc release for now.